storm-client/src/jvm/org/apache/storm/topology/TopologyBuilder.java

```diff
@@ -316,6 +322,68 @@ public BoltDeclarer setBolt(String id, IWindowedBolt bolt, Number parallelism_hi
     }
 
     /**
+     * Define a new bolt in this topology. This defines a lambda basic bolt, which is a
+     * simpler to use but more restricted kind of bolt. Basic bolts are intended
+     * for non-aggregation processing and automate the anchoring/acking process to
+     * achieve proper reliability in the topology.
+     *
+     * @param id the id of this component. This id is referenced by other components that want to consume this bolt's outputs.
+     * @param biConsumer lambda expression which is the instance of functional interface BiConsumer
```

Comment:
Nitpick: This comment doesn't really say much about what the consumer is/does. Maybe something like "lambda expression that implements tuple processing for this bolt"?